apps/connect/src/env/token-bridge.mainnet.ts

```diff
@@ -235,7 +235,7 @@ export const ENV: Env = {
           icon: "https://assets.coingecko.com/coins/images/35087/standard/womrhole_logo_full_color_rgb_2000px_72ppi_fb766ac85a.png?1708688954",
           coinGeckoId: "wormhole",
           decimals: {
-            Ethereum: 18,
+            Ethereum: 6,
```

Comment:
This change could give erros in other cases, the solution should be modify the logic in this code https://github.com/wormhole-foundation/wormhole-connect/blob/5aa4b55c38b836da65e965111e2ab7278854275d/wormhole-connect/src/utils/index.ts#L116-L134, so that in this case (a token bridge route not an ntt https://wormholescan.io/#/tx/1/ec7372995d5cc8732397fb0ad35c0121e0eaa90d26f828a534cab54391b3a4f5/920703?view=advanced), instead of taking the Ethereum value, it takes the default value